Civi/Share/CiviMRF/Connector.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Civi\Share\CiviMRF;
+
+use Civi\Api4\ShareNodePeering;
+
+class Connector {
```

Comment:
What is the purpose of this class? There are two private attributes which aren't used...